src/Error/JsonApiExceptionRenderer.php

```diff
@@ -97,6 +97,10 @@ public function validation($exception)
         // set data and send response
         $this->controller->response->type('jsonapi');
         $this->controller->response->body($json);
+        $this->controller->response->cors($this->controller->request)
```

Comment:
But the ozee31/Cors plugin is overload by crud plugin and return headers have an difference. I think  this crud plugin dont needed cors enabled ? this wold be possible if config on enable plugin, what do you think about this ?